apps/angular/src/app/features/anime/anime-details/anime-details.component.html

```diff
@@ -0,0 +1,41 @@
+<main class="anime-details" *ngIf="animeDetails$ | async as animeDetails">
+	<h1 class="anime-details-title">{{animeDetails.englishTitle ? animeDetails.englishTitle : animeDetails.japaneseTitle}}</h1>
+	<section class="anime-card">
+		<img
+			class="anime-card__image"
+			[src]="animeDetails.imageUrl"
+			alt=""
+		/>
+		<button type="button"
+			class="anime-card__image-full-size-btn" 
+			(click)="openFullSizeImage(animeDetails.imageUrl)">
+			<i class="anime-card__magnifying-glass"></i>
+		</button>
+		<div class="anime-card__info">
+			<h2 class="anime-details-title">About anime</h2>
+			<camp-anime-info-item itemTitle="English title" [itemValue]="animeDetails.englishTitle | titlecase | empty" />
+			<camp-anime-info-item itemTitle="Japanese title" [itemValue]="animeDetails.japaneseTitle | titlecase | empty" />
+			<camp-anime-info-item itemTitle="Aired start date" [itemValue]="animeDetails.airedStartDate | date | empty" />
+			<camp-anime-info-item itemTitle="Type" [itemValue]="animeDetails.type | titlecase" />
+			<camp-anime-info-item itemTitle="Status" [itemValue]="animeDetails.status | titlecase" />
+			<camp-anime-info-item itemTitle="Age rating" [itemValue]="animeDetails.ageRating" />
+			<camp-anime-info-item itemTitle="Source" [itemValue]="animeDetails.source" />
+			<camp-anime-info-item itemTitle="Season" [itemValue]="animeDetails.season" />
+			<camp-anime-info-item itemTitle="Airing status" [itemValue]="animeDetails.isAiring ? 'Yes' : 'No'" />
+			<camp-anime-info-item itemTitle="Studios" [itemValue]="animeDetails.studioNames | commaJoin | empty" />
+			<camp-anime-info-item itemTitle="Genres" [itemValue]="animeDetails.genreNames | commaJoin | empty" />
+		</div>
+	</section>
+	<section class="anime-details__description" *ngIf="animeDetails.description !== ''" >
+		<h2 class="anime-details-title">Description</h2>
+		<p>{{ animeDetails.description }}</p>
+	</section>
+	<section class="anime-trailer anime-details__trailer" *ngIf="animeDetails.youtubeTrailerId !== null">
+		<h2 class="anime-details-title">Youtube trailer</h2>
+		<iframe
+			title="Youtube trailer"
+			class="anime-trailer__player"
+			[src]="makeYoutubeTrailerUrl(animeDetails.youtubeTrailerId)"
+		></iframe>
```

Comment:
It looks like we can create a separate component for displaying YouTube video.
In this case we can reuse it in other places of our app. Besides, we can encapsulate logic related to YouTube url building in such component.
```suggestion
		<camp-youtube-player [videoId]="animeDetails.youtubeTrailerId" />
```